lib/internal/crypto/keys.js

```diff
@@ -639,8 +639,8 @@ function createPrivateKey(key) {
   return new PrivateKeyObject(handle);
 }
 
-function isKeyObject(key) {
-  return key instanceof KeyObject;
+function isKeyObject(obj) {
+  return obj != null && obj[kHandle] !== undefined;
```

Comment:
`kHandle` is used by lots of crypto classes.

Reply:
Use `kKeyType` instead then?

Reply:
✅ and added a test.